python_modules/libraries/dagster-airflow/dagster_airflow/dagster_job_factory.py

```diff
@@ -249,3 +253,106 @@ def make_airflow_example_dags():
     return make_dagster_definitions_from_airflow_dag_bag(
         dag_bag=dag_bag, use_ephemeral_airflow_db=use_ephemeral_airflow_db
     )
+
+def make_graph_backed_assets_from_airflow_dag(
+    dag,
+    asset_key_to_task_ids=None,
```

Comment:
Assuming this is a `Mapping[AssetKey, str]`, our typical naming scheme for this is like `task_ids_by_asset_key`

Reply:
Separately, do we want to provide some way for people to say that some task inputs are provided by assets outside of the job?

Reply:
> do we want to provide some way for people to say that some task inputs are provided by assets outside of the job?

good point, i think we do

Reply:
added a `upstream_asset_keys_by_task_id` param to do this